ascify art/Ascify-Art.py

```diff
@@ -561,7 +591,7 @@ def export():
     if sequence is False:
         # single image save
         operation()
-        exported_file = os.path.join(dir_,var.get()+"."+format_.get())
+        exported_file = os.path.join(dir_, f"{var.get()}.{format_.get()}")

```

Comment:
Function `export` refactored with the following changes:

- Use f-string instead of string concatenation [x5]
- Move assignments closer to their usage